examples/wasm_modules/php-scripts/drupal-patch/.htaccess

```diff
@@ -0,0 +1,180 @@
+#
```

Comment:
[_Not a blocker for the MR_] Should this file be part of this MR? I see we're mixing here several features into a single one 🤔.

Reply:
Yes, you are right. I was focused on Drupal's demo when found the issue with the binaries (#35 ). I'm open to discussing whether to split the MR or not.